src/lifx/utils.js

```diff
@@ -125,20 +186,22 @@ utils.rgbHexStringToObject = function(rgbHexString) {
   };
 };
 
+/**
+ * find mininum number in an array
+ * @param {Array} array of numbers
+ * @return {Number} minium of the given array
+ */
 utils.minNumberInArray = function(array) {
-  const sortedCopy = array.slice();
-  sortedCopy.sort(function(a, b) {
-    return a - b;
-  });
-  return sortedCopy[0];
+  return Math.min.apply(undefined, array);
```

Comment:
Can't we just do `...array` instead or is this a Node.js compatibility thing?
```suggestion
  return Math.min(...array);
```